internal/resources/policy/kind/quota/constants.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package policykindquota
+
+import (
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
+	reciperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/quota/recipe"
+)
+
+const (
+	ResourceName     = "tanzu-mission-control_namespace_quota_policy"
```

Comment:
why is this called namespace quota policy

Reply:
The name of the policy in API is set as type `namespace-quota-policy`

<img width="247" alt="Screenshot 2023-02-03 at 3 10 00 PM" src="https://user-images.githubusercontent.com/32017029/216566485-8bef1ed9-c1e0-4dd2-86cf-f9902565abbb.png">

Reply:
It's seems better to stick with just `quota-policy`?

Reply:
I followed the same convention as API, they are using namespace-quota-policy, even in policy-engine, they have named it as namespace-quota-policy where schemas have been described. For user interface, I have used `Quota policy` in documentation.

Reply:
Is the quota policy is applied only for namespace level resources ? 

I feel just quota policy should be sufficient, but just check with Pranav once. 
What does the UI say ? follow the terminology used in UI. 
Like sourceSecret to repository_credentials

Reply:
Quota policy is applied to org, cluster group and cluster level only. UI says Quota policy but request/ response from API has `namespace-quota-policy`. Have asked Pranav too.

Reply:
Then lets go with the UI and call it Quota policy

Reply:
Done